mismi-s3/test/Test/IO/Mismi/S3/Commands.hs

```diff
@@ -89,9 +89,9 @@ prop_upload_overwrite d1 d2 l = withLocalAWS $ \p a -> do
   let t = p F.</> localPath l
   liftIO . D.createDirectoryIfMissing True $ F.takeDirectory t
   liftIO $ T.writeFile t d1
-  uploadWithMode Fail t a
+  uploadWithModeOrFail Fail t a
```

Comment:
I'm all for using the testing the `uploadWithMode` result instead, one of the many benefits of returning data :)